app/controllers/goods_nomenclatures_controller.rb

```diff
@@ -8,7 +8,12 @@ def goods_code_id
   end
 
   def find_relevant_goods_code_or_fallback
-    @search = Search.new(q: goods_code_id, as_of: @tariff_last_updated)
+    @search = Search.new(
+      q: goods_code_id,
+      'day' => @tariff_last_updated.try(:day),
+      'month' => @tariff_last_updated.try(:month),
+      'year' => @tariff_last_updated.try(:year),
```

Comment:
doesn't work with symbols?

```
day: @tariff_last_updated.try(:day),
month: ...,
year: ...
```

Reply:
We could move the api entity to HashWithIndifferentAccess. The answer is no, though